src/components/connect-hoc.tsx

```diff
@@ -17,22 +18,25 @@ export interface IHOCProps {
 
 function ConnectHOC(opts?: IHOCProps | ((_) => IHOCProps)) {
   return (Comp: any) =>
-    class extends Component {
-      constructor(props) {
-        super(props);
+    hoistStatics(
```

Comment:
Small nit; I'd prefer to name the class and to wrap it (i.e. call `hoistStatics`) on return.

Should clean this file up a little tbh 😅